src/utils/ScorecardRules.ts

```diff
@@ -13,14 +13,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { isNil } from 'lodash';
 import {
   ApplicableRuleOutcome,
   NotApplicableRuleOutcome,
   NotEvaluatedRuleOutcome,
+  Rule,
+  RuleFilter,
   ruleName,
   RuleOutcome,
   RuleOutcomeType,
+  ScorecardLevelRule,
 } from '../api/types';
+import { isNotNullOrEmpty } from './strings';
+
+export interface RuleDetail {
+  id: number | string;
```

Comment:
🙈

Reply:
Yes, I know.. there is a lot of inconsistency in types so I had to define it this way.
Maybe it is worth to create a task to do some types cleanup in plugin.